bin/github-utils.js

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env node
+import {Octokit} from '@octokit/rest'
+import * as fs from 'fs'
+import * as path from 'path'
+import {spawn} from 'child_process'
+import {runCommand} from './run-command.js'
+import {createPullRequest} from 'octokit-plugin-create-pull-request'
+
+/**
+ * @param {string} output
+ * @returns {string}
+ */
+function extractPassword(output) {
+  const passwordRegex = /Password: (\w+)/
+  const match = output.match(passwordRegex)
+  if (match && match[1]) {
+    return match[1]
+  }
+  throw new Error('Password not found in output')
+}
+
+/**
+ * @returns {Promise<string>}
+ */
+async function getGithubPasswordFromDev() {
+  try {
+    // Uses token from `dev`
+    const output = await runCommand('/opt/dev/bin/dev', ['github', 'print-auth'])
+    const password = extractPassword(output)
+    return password
+  } catch (error) {
+    console.warn(`Soft-error fetching password from dev: ${error.message}. Try running \`dev github print-auth\` manually.`)
+    process.exit(0)
+  }
+}
+
+/**
+ * @param {string} owner
+ * @param {function(import('@octokit/rest').Octokit): Promise<boolean>} func
+ * @returns {Promise<boolean>}
+ */
+export async function withOctokit(owner, func) {
```

Comment:
I know you just moved this function, but what about removing the owner param to simplify? It seems we only use it with "shopify"

Reply:
Actually we also use it for "shop", you can't see it in this PR, but if you look in the `get-graphql-schemas.js` file you'll see it

Reply:
Ah! 🤐